core/iwasm/common/wasm_runtime_common.c

```diff
@@ -3216,7 +3216,7 @@ wasm_runtime_init_wasi(WASMModuleInstanceCommon *module_inst,
         goto fail;
     }
     addr_pool_inited = true;
-
+#if !defined(BH_PLATFORM_ZEPHYR) 
```

Comment:
Could you implement os_convert_stdin_handle, os_convert_stdout_handle, os_convert_stderr_handle and some other APIs if needed? It is strange that these critical codes are commented for zephy.